comp110-worksheet-6/OxoBoard.cs

```diff
@@ -10,41 +10,120 @@ public enum Mark { None, O, X };
 
 	public class OxoBoard
 	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+        public Mark[,] gameBoard;
+
+        /// <summary>
+        /// Using the height and width parameters to create the new board size.
+        /// </summary>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        /// <param name="inARow"></param>
```

Comment:
If you choose to use `///` comments (a good idea in general) you should make proper use of the `param` and `return` fields to describe the parameters and return value